modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/WindowStage.java

```diff
@@ -152,73 +139,70 @@ public final WindowStage init(GlassSystemMenu sysmenu) {
     private void initPlatformWindow() {
         if (platformWindow == null) {
             Application app = Application.GetApplication();
-            if (isPrimaryStage && (null != appletWindow)) {
-                platformWindow = app.createWindow(appletWindow.getGlassWindow().getNativeWindow());
-            } else {
```

Comment:
The diffs in this method below this point are mostly caused by indentation. I recommend reviewers select the "Hide whitespace changes" option.